ackhandler/interfaces.go

```diff
@@ -12,6 +12,7 @@ type SentPacketHandler interface {
 	// SentPacket may modify the packet
 	SentPacket(packet *Packet) error
 	ReceivedAck(ackFrame *wire.AckFrame, withPacketNumber protocol.PacketNumber, recvTime time.Time) error
+	HandshakeComplete()
```

Comment:
Nit: SetHandshakeComplete, as non-"const" functions should start with a verb.

Reply:
Done.